apps/parcel/src/app/app.module.ts

```diff
@@ -0,0 +1,12 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { ParcelModule } from 'single-spa-angular/parcel';
```

Comment:
Why don't we have to import ParcelComponent here and add it to the declarations block?